examples/parameters/main.go

```diff
@@ -116,3 +118,49 @@ func taskComplexParam(flow *goyek.Taskflow) goyek.Task {
 		},
 	}
 }
+
+type listParamValue []int
+
+func (value *listParamValue) Set(s string) error {
+	v, err := strconv.Atoi(s)
+	if err != nil {
+		err = errors.New("parse error")
+	}
+	*value = append(*value, v)
+	return err
+}
+
+func (value *listParamValue) Get() interface{} {
+	return []int(*value)
+}
+
+func (value *listParamValue) String() string {
+	return fmt.Sprintf("%v", *value)
+}
+
+func (value *listParamValue) IsBool() bool {
+	return false
+}
+
+// taskListParam showcases repeatable parameters.
+//
+// Execute `go run ./main.go -v list -port 1 -port 2 -port 3` as an example.
```

Comment:
nit:
```suggestion
// Execute `go run . -v list -port 1 -port 2 -port 3` as an example.
```

applies to other comments as well